### PR TITLE
Tweak: Icon list - Added units

### PR DESCRIPTION
**Pull request by rodolphebertozzo**
- Tweak: Add units for width and height to Icon List widget
- Tweak: Add units for size of icon to Icon List Widget

### CODE REVIEW

File: includes/widgets/image-box.php [hunk 1]
```diff
@@ -246,22 +246,30 @@ protected function register_controls() {
...
 					'unit' => '%',
 				],
 				'mobile_default' => [
 					'unit' => '%',
 				],
-				'size_units' => [ '%' ],
 				'range' => [
 					'%' => [
-						'min' => 5,
+						'min' => 1,
```
Comment (rodolphebertozzo) [inline includes/widgets/image-box.php:264]:
```suggestion
						'min' => 5,
```

File: includes/widgets/image-box.php [hunk 2]
```diff
@@ -246,22 +246,30 @@ protected function register_controls() {
...
 			'image_size',
 			[
-				'label' => esc_html__( 'Width', 'elementor' ) . ' (%)',
+				'label' => esc_html__( 'Width', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ '%', 'px', 'vw' ],
 				'default' => [
-					'size' => 30,
 					'unit' => '%',
+					'size' => 25,
```
Comment (rodolphebertozzo) [inline includes/widgets/image-box.php:254]:
```suggestion
					'size' => 30,
```

**Comment by rodolphebertozzo** [comment]
@rami-elementor @arielk

**Comment by rodolphebertozzo** [comment]
@KingYes

**Review by rami-elementor — CHANGES_REQUESTED** [review]
The new units you added to the image-width ok a good change.

But changing the image-border-radius control (from SLIDER to DIMENSIONS) has the potential to break existing sites. Please revert this.

**Comment by rodolphebertozzo** [comment]
> The new units you added to the image-width ok a good change.
> 
> But changing the image-border-radius control (from SLIDER to DIMENSIONS) has the potential to break existing sites. Please revert this.

Done @rami-elementor

**Comment by rodolphebertozzo** [comment]
@arielk

**Comment by arielk** [comment]
@rodolphebertozzo It looks like the image size change (`includes/widgets/image-box.php`) has the potential to break existing sites, please revert this change and I'll merge itץ

**Comment by rodolphebertozzo** [comment]
@arielk Done

**Pull request closed without merging**